### ISSUE TITLE
App unusable on macOS 13.0.1 (works fine on 13.0)

### ISSUE THREAD

**Issue by ddotlic** [keeweb/keeweb#2067]
**Describe the bug**
Fatal crash on macOS 13.0.1 arm64 (all good on 13.0); _correction_, does **not** fail on an Intel MacBook Pro.

@antelle  I just saw the pinned issue and am sad for your personal situation, whatever it may be. You don't owe us anything and yeah, it would be a shame to let this project rot. In this particular instance, I am (educated guess!) almost sure the issue is in the Electron Shell; if you can, please answer a question below, no need to work on anything?
Can anyone build the product as-is? IOW, if I managed to upgrade to a newer Electron, should I be able to build the app - no special "keys" or certs necessary?  (obviously, if the patch is good, I _will_ contribute to the repo for everyone)?

**To Reproduce**
1. Start the app
2. The app starts fine and you can go into settings, for example
3. As soon as password is entered correctly, a fatal crash.

**Expected behavior**
The app doesn't crash 😉 

**Screenshots**
N/A

**Environment**
macOS 13.0.1, details in the crash log below

**Kdbx File**
N/A, any file should be problematic

**Logs**

[crash.log](https://github.com/keeweb/keeweb/files/9989517/crash.log)

**Comment by Tobisaninfo** [comment]
Having the same issue with macOS 13.1. Has anyone an idea how to fix this issue?

**Comment by ddotlic** [comment]
> Has anyone an idea how to fix this issue?

I managed to make it work; the clue was installing the app on a new computer where it somehow worked, so I knew that the crash was avoidable.

1. You _may_ want to ignore the prompt to protect the app from being writable by an admin account - I am not sure if this makes a difference, just an FYI (would be better to keep the protection on of course)
2. You **definitely** want to trash the app's settings; for reference, they are stored at `~/Library/Application\ Support/KeeWeb/`

Verified to work on a Studio with M1 and an Air M2.
